scripts/utils.ps1

```diff
@@ -114,3 +114,10 @@ function Test-FileExists([string]$path) {
         throw "Could not find '${path}'."
     }
 }
+
+function CreateFolder([string] $folderName){
```

Comment:
May be RecreateFolder? Seems like it matches the behavior better.

Reply:
I wouldn't rename this since the goal is to create the folder. It's named similar with [File.Create](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.create?view=net-5.0) which overrides the file if it exists.